src/main/kotlin/lotto/domain/LottoTickets.kt

```diff
@@ -6,5 +6,17 @@ class LottoTickets(val lottoTickets: List<LottoTicket>) {
         fun from(lottoTickets: List<LottoTicket>): LottoTickets {
             return LottoTickets(lottoTickets)
         }
+
+        fun getWinningTicket(inputUserTicket: LottoTickets, luckyLottoTicket: LottoTicket, bonusBallNumber: LottoNumber): List<Rank> {
+            var rankList = mutableListOf<Rank>()
+
+            inputUserTicket.lottoTickets.forEach {
+                var matchCount = it.getMatchValueCount(luckyLottoTicket)
+                val isBonusBall = it.isExistBonusBall(bonusBallNumber)
+                val rank: Rank = Rank.of(matchCount, isBonusBall)
+                rankList.add(rank)
+            }
+            return rankList
```

Comment:
1. MutableCollection 생성
2. MutableCollection에 추가
3. 불변 Collection형으로 반환

위 3단계는 다음과 같이 `map`으로 간단하게 작성할 수 있습니다.
```suggestion
            return inputUserTicket.lottoTickets.map  {
                val matchCount = it.getMatchValueCount(luckyLottoTicket)
                val isBonusBall = it.isExistBonusBall(bonusBallNumber)
                Rank.of(matchCount, isBonusBall)
            }
```